pkg/task/v1alpha1/runtask_command.go

```diff
@@ -400,16 +416,16 @@ type WillRunFn func() bool
 //
 // 2/ 'store' hook can be used to save runtask command's execution response
 type defaultCommandRunner struct {
-	store   ResultStoreFn // mechanism to store runtask command's response
-	willrun WillRunFn     // flag that determines if runtask command will execute or not
-	id      string        // unique identification of the runtask command
-	cmd     *RunCommand   // manages execution of runtask command
-	*Msgs                 // store and retrieve info, warns, errors, etc occurred during execution
+	store     ResultStoreFn // mechanism to store runtask command's response
+	willrun   WillRunFn     // flag that determines if runtask command will execute or not
+	id        string        // unique identification of the runtask command
+	cmd       *RunCommand   // manages execution of runtask command
+	*msg.Msgs               // store and retrieve info, warns, errors, etc occured during execution
```

Comment:
`occurred`

Reply:
@princerachit you want me to change `msg` to `occurred` ?

Reply:
No... just cahgne `occured` to `occurred`

Reply:
Updated